test_util/test_upgrade_vpc.py

```diff
@@ -144,14 +144,14 @@ def main():
 
     master_list = [h.private_ip for h in cluster.masters]
 
-    cluster_api = ClusterApi(
+    cluster_api = DcosApiSession(
         'http://{ip}'.format(ip=cluster.masters[0].public_ip),
         master_list,
         master_list,
         [h.private_ip for h in cluster.agents],
         [h.private_ip for h in cluster.public_agents],
```

Comment:
this corresponds to 'slaves' that is typed as 'str'. I guess, that should be list in DcosApiSession.